### DISCUSSION TITLE
Gollum only opens 4567/gollum/create/Home and not my wiki

### DISCUSSION THREAD

**Discussion by K-Takouti** [gollum/gollum#2035] [Q&A] [answered]
HELP PLEASE, NEWBIE HERE, 

- I Downloaded Ruby as a Windows user.  C:\Ruby32-x64
- ran:  gem install gollum in cmd.  So it got installed somewhere in C:\Ruby32-x64 in bin and lib, I guess
- Then Went on Git bash on my Git Repository : ~/Desktop/test-GitLab.wiki (main) and ran : 
    + gollum 
    + gollum --path /c/Users/Me/Desktop/test-GitLab.wiki
    + every thing that ChatGPT would suggest (changing config.rb, clearing cache, changing browser, ...)
    
->  Every time it opens http://localhost:4567/gollum/create/Home 

I need it to open Desktop/test-GitLab.wiki, but I am just not getting it. Please Help

**Comment by benjaminwil** [comment, marked as answer]
Hey there. I think what you've described is the expected behaviour of Gollum.

`Home` is the "index" page that Gollum opens if you don't specify a page to another document.

If your wiki does not have this index file in the root directory, `Home.md`, then Gollum will route you to the create page so you can create one.

To further troubleshoot your issue, I recommend going ahead and creating this `Home` page and then clicking the "Overview" link. You should be able to see the other pages in your `test-GitLab.wiki` directory.

Hope this helps.
